test/testCLI.ts

```diff
@@ -82,4 +82,15 @@ export function run() {
     unlinkSync('./ReferencedType.d.ts')
   })
 
+  test('files in (-i), files out (-o)', t => {
```

Comment:
This looks great! Would you mind also adding test cases for:

- Input directory specified, output directory not specified
- Input directory specified, output directory is specified and nested, and does not exist (/foo/bar/baz)
- Input directory is specified,